dbt/models/marts/candidats_enriched.sql

```diff
@@ -1,16 +1,16 @@
 select
     {{ pilo_star(ref('stg_candidats')) }},
     case
-        when candidats.age_selon_nir < 25 then 'jeune (-25 ans)'
-        when candidats.age_selon_nir > 25 and candidats.age_selon_nir < 55 then 'adulte (25-55 ans)'
-        when candidats.age_selon_nir > 55 then 'senior (+55 ans)'
+        when candidats.age_selon_nir <= 25 then 'jeune (- de 26 ans)'
+        when candidats.age_selon_nir > 25 and candidats.age_selon_nir <= 54 then 'adulte (26-54 ans)'
```

Comment:
TIL tu pourrais faire `between 25 and 54` mais c'est inclusif et pas moyen de dire qu'une des deux bornes doit être exclusive. Du coup utiliser `between` est plus confusant qu'autre chose, dommage. On oublie.